explore-analyze/discover/discover-get-started.md

```diff
@@ -22,6 +22,26 @@ Learn how to use **Discover** to:
 * You must have data in {{es}}. Examples on this page use the [ecommerce sample data set](../index.md#gs-get-data-into-kibana), but you can use your own data.
 * You should have an understanding of [{{es}} documents and indices](../../manage-data/data-store/index-basics.md).
 
+## Context-aware data exploration [context-aware-discover]
+
+**Discover** can provide tailored experiences for the following data types when accessed from specific Serverless project types or {{kib}} solution views:
+
+* Observability:
+  * **[Logs exploration](/solutions/observability/logs/explore-logs.md)**
+% LINK/PAGE TBD  * **Traces exploration**
+% LINK/PAGE TBD  * **Metrics exploration**
+% * Security: 
+% LINK/PAGE TBD  * **Security data exploration**
+
+This context-aware experience is determined by both your solution context and the type of data you query. When both conditions align, **Discover** provides specific capabilities useful for exploring that specific type of data, and integrates features or paths to other relevant solution applications.
```

Comment:
```suggestion
What appears is determined by both your solution context and the type of data you query. When both conditions align, **Discover** provides specific capabilities useful for exploring that specific type of data, and integrates other relevant features.
```
If this is still accurate, it would be a bit simpler.